client/src/components/organisms/Header/Header.jsx

```diff
@@ -1,19 +1,28 @@
 import React from 'react';
 import { Navbar, Nav, Button } from 'react-bootstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faRoute, faPlus } from '@fortawesome/free-solid-svg-icons';
+
 import LoginModal from '../LoginModal/LoginModal';
 
-const Header = () => {
-  return (
-    <Navbar bg='light' expand='lg'>
-      <Navbar.Brand href='/'>Tripen</Navbar.Brand>
-      <Navbar.Toggle aria-controls='basic-navbar-nav' />
-      <Navbar.Collapse id='basic-navbar-nav'>
-        <Nav className='ml-auto'>
-          <Button variant='primary'>Create Timeline</Button>
-          <LoginModal></LoginModal>
-        </Nav>
-      </Navbar.Collapse>
-    </Navbar>
-  );
-};
+const Header = () => (
+  <Navbar bg="light" expand="lg">
+    <Navbar.Brand href="/">
+      <FontAwesomeIcon icon={faRoute} />
+      {' '}
+      Tripen
```

Comment:
I think now is a good time to replace 'Tripen' with a logo (component). Maybe just place in the react logo from CRA for now.